### ISSUE TITLE
Lack of synchronization between kuery bars on Overview/Monitor Status alert

### ISSUE THREAD

**Issue by justinkambic** [elastic/uptime#224]
**Describe the bug**
When the user updates the kuery bar in the alerting flyout from within the Overview page, the kuery bar behind the flyout doesn't update to reflect the changes. This leads to a confusing UI state when the user clicks out of the flyout, because the rest of the app reflects the updated state, while the kuery bar still shows outdated filters.

![Jun-18-2020 12-47-34](https://user-images.githubusercontent.com/18429259/85049048-34dc8500-b162-11ea-987e-1127820411ce.gif)

**To Reproduce**
1. Open the Overview page
2. Specify a kuery
3. Open the monitor status alert
4. Change the value in the kuery bar
5. Exit the alert flyout
6. The kuery bar should not match the rest of the values displayed in the UI

**Expected behavior**
When the kuery bar in the alert is re-submitted with new values, the value shown in the other bar should also change to match the new parameter.

**Debug logs**

**Comment by andrewvc** [comment]
Candidly, there's something weird about these things being synchronized. If the flyout didn't hide the background with a translucent div I'd say yeah, syncing makes sense, but given that it does it's weird that we expect the flyout to change the obscured background.

@katrin-freihofner WDYT here UI-wise? I kind of think it should be independent or we should come up with some other solution if the alert requires the background page.

**Comment by katrin-freihofner** [comment]
I agree. The view in the background should not be updated. Didn't we once discuss to have a hint how many monitors will be affected by this filter?

We could also do something similar to what the Metrics app does:
<img width="609" alt="Screenshot 2020-07-22 at 09 22 54" src="https://user-images.githubusercontent.com/20965076/88147068-00defe80-cbfd-11ea-8e30-b80c732f3a66.png">

**Comment by justinkambic** [comment]
@andrewvc the simplest solution then is to treat this as a completely different piece of state from the overview page (which would probably simplify this component as well) and not have the flyout impact the overview page selection. It would enable us to store the flyout's selections entirely in local state, and provide defaults from the overview page, if they're available.

@katrin-freihofner the alert flyout in Uptime does provide a hint in a callout now, similar to the metrics example.

**Comment by andrewvc** [comment]
@justinkambic I think that's the right decision. The searches backing the two pages are actually a bit different, and ultimately entangling these two things is more trouble than it's worth. From the UI side, I think the right answer is to go back to masking the overview page while the flyout is active so people aren't mislead.

Thoughts @katrin-freihofner @drewpost ?

**Comment by drewpost** [comment]
I agree - these should not be synced. Mask it up!

**Comment by paulb-elastic** [comment]
As discussed in the design syn, it is ok that modify the kuery in the alert box should not change the kuery on the main overview page, however the _background_ overview page should be faded so that the alert overlay is more prominent and visually disconnected from the overview page.

**Comment by justinkambic** [comment]
Closing this as Uptime's deprecated and we haven't given this any thought in many years.
